src/containers/NFT/NFTHeader/NFTHeader.tsx

```diff
@@ -48,7 +64,40 @@ const NFTHeader = (props: Props) => {
     }
   }, [tokenId])
 
-  const data = rawData && formatNFTInfo(rawData)
+  const data = useMemo(() => rawData && formatNFTInfo(rawData), [rawData])
```

Comment:
Why is this needed?

Reply:
I added it to avoid re-computing the format function, because this component has several api responses that will re-render. I can remove it I dont think it has any noticeable difference